test/test-tetris.js

```diff
@@ -1088,3 +1088,139 @@ describe("generation Square Tetramino", function(){
 		assert.deepEqual(transformation, bodyTetramino);	
 	})
 })
+
+describe("L-tetramino rotation 4x4", function() {
+	let field = tetris.buildField(`
+		----
+		----
+		----
+		----
+	`);
+	
+	it('After 4 turns on an empty field, the L-tetramino should remain in the same place where it was;', function(){
+		let L_Tetramino = tetris.buildTetramino(`
+			----
+			--X-
+			XXX-
+			----
+		`);
+		let rotated90_L_Tetramino = tetris.rotateTetramino_L(field, L_Tetramino);
```

Comment:
I want `tetris.rotateTetramino` to handle any kind of tetromino
So you don't need 6 different functions to do rotations etc.